React/Base/RCTUtils.m

```diff
@@ -829,11 +829,28 @@ BOOL RCTIsLocalAssetURL(NSURL *__nullable imageURL)
   return RCTImageFromLocalAssetURL(bundleImageUrl);
 }
 
+#if TARGET_OS_OSX // [TODO(macOS GH#774)
+static NSCache<NSURL *, UIImage *> *RCTLocalImageCache()
+{
+  static NSCache *imageCache;
+  static dispatch_once_t onceToken;
+  dispatch_once(&onceToken, ^{
+    imageCache = [NSCache new];
```

Comment:
we should definitely limit this right? it can be abused badly ? also it seems like we should also clear out once in a while?

Reply:
Looks like he added a 100 file limit